.github/workflows/ci.yml

```diff
@@ -41,21 +44,84 @@ jobs:
           pre-commit run --all-files
           pytest taf/tests
 
+  build_and_upload_wheel:
+    runs-on: ubuntu-latest
+    needs: run_tests
+    if: github.event_name == 'release'
```

Comment:
Should this be on release or on tag?

Reply:
We typically release TAF by drafting a new release. We also need the release URL to upload the assets